pkg/config/validation.go

```diff
@@ -36,3 +42,67 @@ func Validate(pArgs *ProgArgs) error {
 
 	return nil
 }
+
+func validateConfigLetPath(configletDir, configletName string) (string, error) {
+	configletPath, err := filepath.EvalSymlinks(filepath.Clean(filepath.Join(configletDir, configletName)))
```

Comment:
`EvalSymlinks` failed if file doesn't exist.  
Could this be problematic for testing?

Reply:
yes, this is the reason for https://github.com/k8stopologyawareschedwg/resource-topology-exporter/pull/304/files#diff-27fc28a543a4903aa873a125029a1f3ae2833f08470eae1c3c64b55e222810acR143